docs/angular/api-web/executors/dev-server.md

```diff
@@ -26,6 +26,14 @@ Type: `string`
 
 Target which builds the application
 
+### hmr
+
+Default: `false`
```

Comment:
Can we enable by default so users will get the better experience out of the box?

And they can use `--no-hmr` or `--hmr false` to disable it.